setup.py

```diff
@@ -115,7 +119,8 @@ def extensions():
     from Cython.Build import cythonize
     return cythonize(Extension('arctic._compress',
                                sources=["src/_compress.pyx", "src/lz4.c", "src/lz4hc.c"],
-                               extra_compile_args=['-fopenmp', '-fpermissive'], # Avoid compiling error with prange. Similar to http://stackoverflow.com/questions/36577182/unable-to-assign-value-to-array-in-prange
+                               # Avoid compiling error with prange. Similar to http://stackoverflow.com/questions/36577182/unable-to-assign-value-to-array-in-prange
```

Comment:
Presumably this should be hoisted up to line 30 where `-fpermissive` is defined?

Reply:
Otherwise LGTM!

Reply:
Moved the comment to be by the flag.